doc/operation-and-maintenance/gdpr-considerations.md

```diff
@@ -1,18 +1,18 @@
 # GDPR considerations
 
-This page describes what GDPR implies in terms of the server management.
+This page describes what GDPR implies in terms of server management.
 
 ## Data affected by GDPR commands
 
-* inbox - All entries in subject's inbox. If their messages are stored in other users' inbox, they will not be removed.
+* inbox - All entries in the subject's inbox. If their messages are stored in other users' inbox, they will not be removed.
 * message archive - Same as above for 1-1 messages. In case of group chat messages, they are retrieved as personal data but not removed.
 * offline storage - All messages stored for delivery.
-* roster - All entries in user's roster. Other users' rosters are NOT affected, even if they include subject's JID or other data.
-* vCard - The entire content of user's vCard.
-* private XML storage - All items stored by the client will be removed.
+* roster - All entries in the subject's roster. Other users' rosters are NOT affected, even if they include the subject's JID or other data.
+* vCard - The entire content of the subject's vCard.
+* private XML storage - All items stored by the subject will be removed.
 * publish-subscribe
-    * retrieval: User's all payloads, nodes created by them and all of their subscriptions.
-    * removal: User's subscriptions, push and PEP nodes (with their data included).
+    * retrieval: all subject's payloads, nodes created by him and all of his subscriptions.
```

Comment:
retrieval: all subject's payloads, nodes created by them and all of their subscriptions.

Reply:
I think the pronouns refer to the subject (singular) who created nodes and subscribed to other nodes, not to the payloads (plural). Otherwise, payloads don't subscribe to things. Perhaps making the two more consistent would help, something like
```
     * retrieval: subject's subscriptions, nodes (with their payloads included). 
     * removal: subject's subscriptions, push and PEP nodes (with their data included). 
```
@fenek?

Reply:
Your seconds proposition sounds good but regarding Kat's comment: https://en.wikipedia.org/wiki/Singular_they

Reply:
Heck I had no idea about that _(silently learns something new every day)_ 😃